File: JavaScript/01.JavaScript-Basic-February-2021/04.conditionalStatements-Exercise/05.time15Minutes.js
function time(input) {
    let h = Number(input[0]);
    let m = Number(input[1]);

    let totalMin = h * 60 + m;
    let totalMinAfter = totalMin + 15;

    let finalH = Math.floor(totalMinAfter / 60);
    if (finalH > 23) {
        finalH = 0;
    }

    let finalM = totalMinAfter % 60;
    if (finalM < 10) {
        console.log(`${finalH}:0${finalM}`);
    }else{
        console.log(`${finalH}:${finalM}`);
    }

}

time(["11", "08"]);

File: Python/01.Python-Basic-April-2019/05.conditional_statements_exercise/05.time_15_minutes.py
import math

hour = int(input())
minutes = int(input())

time_minutes = hour * 60 + minutes
time_after_15_minutes = time_minutes + 15

hour = time_after_15_minutes / 60
minutes = time_after_15_minutes % 60

hour = math.floor(hour)

if hour == 24:
    hour = 0

if minutes < 10:
    print(f'{hour}:0{minutes}')
else:
    print(f'{hour}:{minutes}')
